src/_posts/databases/postgresql/2000-01-01-postgis.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Is this page still a work in progress?

Reply:
Well, for now that's all we have.
The idea was to have a dedicated page to make it a bit more visible to the public and also to have a page ready for potential future information/tips.